Plugins/Sycl/src/Seeding/CreateSeedsForGroupSycl.cpp

```diff
@@ -546,9 +541,24 @@ void createSeedsForGroupSycl(
 
       // Counting the seeds in the second kernel allows us to copy back the
       // right number of seeds, and no more.
-
       seeds.resize(M);
-      std::vector<uint32_t> deviceCountTriplets(edgesBottom, 0);
+      vecmem::vector<uint32_t> countTriplets(&resource);
+      countTriplets.resize(edgesBottom, 0);
```

Comment:
This is more of a PSA for Konrad's benefit, it does not necessarily need to be changed here.

VecMem vectors can still be created with a single line, to be initialised to some size/value. Like:

```c++
vecmem::vector<uint32_t> countTriplets(edgesBottom, 0, &resource);
```

This uses the same `std::vector` constructor ("number 3" from https://en.cppreference.com/w/cpp/container/vector/vector) as the previous incantation used. We just rely on the memory resource pointer being "auto converted" into [std::pmr::polymorphic_allocator](https://en.cppreference.com/w/cpp/memory/polymorphic_allocator).

Just something to keep in mind for future coding... :wink: